test/test_device.py

```diff
@@ -657,11 +657,21 @@ def test_display_address_multisig_descriptor(self):
         sh_wsh_multi_addr = self.wrpc.getaddressesbylabel("shwshmulti-display-desc").popitem()[0]
         wsh_multi_addr = self.wrpc.getaddressesbylabel("wshmulti-display-desc").popitem()[0]
 
-        # need to replace `'` with `h` and to remove checksome for the stdin option to work
+        # need to replace `'` with `h` and to remove checksum for the stdin option to work
         sh_multi_desc = sh_multi_desc.replace("'", "h").split('#')[0]
         sh_wsh_multi_desc = sh_wsh_multi_desc.replace("'", "h").split('#')[0]
         wsh_multi_desc = wsh_multi_desc.replace("'", "h").split('#')[0]
 
+        # descriptor with xpubs
+        if self.full_type == 'trezor_t':
+            account_xpub = self.do_command(self.dev_args + ['getxpub', 'm/48h/0h/0h/2h'])['xpub']
+            desc = 'wsh(multi(2, [' + self.fingerprint + '/48h/0h/0h/2h]' + account_xpub + '/0/0,[' + self.fingerprint + '/48h/0h/0h/2h]' + account_xpub + '/1/0))'
+            result = self.do_command(self.dev_args + ['displayaddress', '--desc', desc])
+            self.assertNotIn('error', result)
+            self.assertNotIn('code', result)
+            self.assertIn('address', result)
+            self.assertEqual(self.wrpc.deriveaddresses(desc)[0], result['address'])
```

Comment:
`deriveaddresses` needs the checksum. Also regtest and testnet have different prefixes. This diff should handle that, and also use BIP 45 (for some reason 48 wasn't working with my emulator).

Don't forget `from hwilib.descriptor import AddChecksum` at the top of `test_device.py`

```suggestion
            account_xpub = self.do_command(self.dev_args + ['getxpub', 'm/45h/0h/0h/2h'])['xpub']
            desc = 'wsh(multi(2,[' + self.fingerprint + '/45h/0h/0h/2h]' + account_xpub + '/0/0,[' + self.fingerprint + '/45h/0h/0h/2h]' + account_xpub + '/1/0))'
            result = self.do_command(self.dev_args + ['displayaddress', '--desc', desc])
            self.assertNotIn('error', result)
            self.assertNotIn('code', result)
            self.assertIn('address', result)
            addr = self.wrpc.deriveaddresses(AddChecksum(desc))[0]
            # removes prefix and checksum since regtest gives
            # prefix `bcrt` on Bitcoin Core while wallets return testnet `tb` prefix
            self.assertEqual(addr[4:58], result['address'][2:56])
```